### ISSUE TITLE
(In Certain Places) The Right Click Menu Does Not Appear on Linux.

### ISSUE THREAD

**Issue by CuriousTommy** [BullyWiiPlaza/JGeckoU#27]
When I tried JGecko U on my Surface, I was able to access the right click menu in the Codes and Memory Viewer section. So I tried to right click on my Linux machine, but the right click menu does not appear at all in either the Codes or Memory Viewer section.

In the Codes section, there is a blue highlight when you select the code (left or right click). This blue highlight is never shown on Linux when you try to left or right click on it (I am not sure if this is important, but I thought I should talk about it just in case).

The weird thing is that if you go to the Add Code window and right clicking in one of the text fields, it would work perfectly fine on Linux.

**Comment by BullyWiiPlaza** [comment]
On Linux, I can replicate that the right click context menu on codes is not working but in text areas it is working. I will see about it.

**Comment by CuriousTommy** [comment]
> Let me know if it all works now.
I can confirm that it works on the Code and Memory Viewer. Thanks Bully!

> How come this took so long to be reported? It's quite significant and some people are using Linux, don't they?
I actually noticed before that right-clicking does not bring up a right click menu. At first, I thought you didn't implement it. But once I asked, on GBATemp, how to poke the values from a search result, I found out from you that there was a right click menu. So I did some testing which led to me making this bug report.

Also, I was too busy with school to do some investigation into this and make a bug report. I, however, am not sure why no one looked into this bug. Maybe I am the only one who really cares about seriously using JGecko U on Linux lol.
